ocs_ci/ocs/ui/base_ui.py

```
@@ -1546,6 +1546,38 @@ def proceed_to_login_console():
+    current_url = driver.current_url
+    endpoint_url = (
+        endpoint if endpoint.startswith("https://") else f"https://{endpoint}"
+    )
+    logger.info("Accepting certificate for S3 endpoint: %s", endpoint_url)
+    driver.get(endpoint_url)
+    time.sleep(2)
+    try:
+        body = driver.find_element(By.TAG_NAME, "body")
+        body.send_keys("thisisunsafe")
+        time.sleep(2)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)